base_layer/core/src/mempool/proto/mod.rs

```diff
@@ -24,7 +24,7 @@
 use crate::transactions::proto::types;
 
 pub mod mempool {
-    tari_utilities::include_proto_package!("tari.mempool");
+    include!(concat!(env!("OUT_DIR"), "/", "tari.mempool.rs"));
```

Comment:
Hope to get rid of the need for `include!` soon - seems to be breaking autocomplete in editors and the `tari_protobuf_build` crate ensures that generated structs are correctly formatted, so we can include these directly in the  module tree.

e.g. 
```rust
#[path = "tari.mempool.rs"]
mod mempool;
```

Reply:
nice